server/src/main/java/server/haengdong/presentation/EventController.java

```diff
@@ -65,13 +65,12 @@ public ResponseEntity<MembersResponse> findAllMembers(@PathVariable("eventId") S
         return ResponseEntity.ok(response);
     }
 
-    @PutMapping("/api/events/{eventId}/members/{memberName}")
+    @PutMapping("/api/events/{eventId}/members/nameChange")
```

Comment:
r: api uri에 PathVariable 이외에 카멜케이스를 쓰지 않고 있었는데요,
```suggestion
    @PutMapping("/api/events/{eventId}/members/name-change")
```
bill-actions, member-actions 처럼 하거나 혹은
```suggestion
    @PutMapping("/api/events/{eventId}/members/change")
```
이렇게 바꾸는게 어떨까요?

Reply:
저도 카멜케이스보다는 name-change 가 좋은 것 같아요.
프론트가 nameChange로 작업중이니 끝나고 바꾸시죠.